src/Apps/Order/Routes/__tests__/Accept.jest.tsx

```diff
@@ -210,22 +209,6 @@ describe("Accept seller offer", () => {
       await page.expectAndDismissDefaultErrorDialog()
     })
 
-    it("commits fixFailedPayment mutation with Gravity credit card id", async () => {
-      commitMutation.mockReturnValue(acceptOfferPaymentRequiresAction)
```

Comment:
**question:** Should we keep a test to assert `it("commits commerceBuyerAcceptOffer mutation again")` if the first attempt resulted in a PaymentRequiresAction error?

Reply:
Yes! It took me some time, but here we have it [3187586](https://github.com/artsy/force/pull/12869/commits/3187586bf973dcf8b74b8865489415b2a6575910)